l1-contracts/bootstrap.sh

```diff
@@ -56,6 +56,9 @@ function test_cmds {
   echo "$hash cd l1-contracts && solhint --config ./.solhint.json \"src/**/*.sol\""
   echo "$hash cd l1-contracts && forge fmt --check"
   echo "$hash cd l1-contracts && forge test --no-match-contract UniswapPortalTest"
```

Comment:
The `--no-match-contract UniswapPortalTest` should not be here when the cahnge was made in `foundry.toml`. 

The change in the toml is applied to forge test calls such that we skip uniswap and the scream and shout test explicitly. This is visible from https://github.com/AztecProtocol/aztec-packages/pull/14414/files

```suggestion
  echo "$hash cd l1-contracts && forge test"
```